DigitalLearningSolutions.Web/Views/Shared/_CookieConsentPartial.cshtml

```diff
@@ -0,0 +1,45 @@
+    var consentViaBannerPostTemp = TempData["consentViaBannerPost"];  // To display confirmation banner (nhsuk-cookie-confirmation-banner-post) after post
+    var path = Context.Request.Path; // @(Configuration["CurrentSystemBaseUrl"]    
+}
+@if (showCookieBanner == null && validateCookieBannerViaTempData == null) // [BY] Show cookie banner when the value is null. if the user consent yes or no then we dont display the banner
+{
+    <div id="nhsuk-cookie-banner">
+        <div class="nhsuk-cookie-banner1" id="cookiebanner">
+            <div class="nhsuk-width-container">
+              <br>
+                <h2>Cookies on the Digital Learning Solutions</h2>
+                <p>We've put some small files called cookies on your device to make our site work.</p>
+                <p>We'd also like to use analytics cookies. These send anonymous information about how our site is used to a service called Google Analytics & Hotjar Analytics. We use this information to improve our site.</p>
+                <p>Let us know if this is OK. We'll use a cookie to save your choice. You can <a id="nhsuk-cookie-banner__link" target="_blank" href="@Url.Action( "CookiePolicy","CookieConsent")" tabindex="1">read more about our cookies</a> before you choose.</p>
+                <p>
+                    <div id="cookie-banner-no-js-styling">
+                        <a href="@Url.Action( "CookieConsentConfirmation","CookieConsent", new { consent="false",path = path})" class="nhsuk-button">Do not use analytics cookies</a>
+                        <a href="@Url.Action( "CookieConsentConfirmation","CookieConsent", new { consent="true",path = path})" class="nhsuk-button">I'm OK with analytics cookies</a>
+                    </div>
+                    <div id="cookie-banner-js-styling" style="display:none;">
+                        <button id="nhsuk-cookie-banner__link_accept" class="nhsuk-button"> Do not use analytics cookies</button>
+                        <button id="nhsuk-cookie-banner__link_accept_analytics" class="nhsuk-button"> I'm OK with analytics cookies</button>
+                    </div>
+                </p>
+            </div>
+        </div>
+
+        <div class="nhsuk-success-banner" id="nhsuk-cookie-confirmation-banner" style="display:none;">
+            <div class="nhsuk-width-container">
+                <p id="nhsuk-success-banner__message">You can change your cookie settings at any time using our <a href="@Url.Action( "CookiePolicy","CookieConsent")">cookies page</a>.</p>
+            </div>
+        </div>
+    </div>
+}
+@if (consentViaBannerPostTemp != null)
+{
+    <div class="nhsuk-success-banner" id="nhsuk-cookie-confirmation-banner-post">
+        <div class="nhsuk-width-container">
+            <p id="nhsuk-success-banner__message">You can change your cookie settings at any time using our <a href="@Url.Action( "CookiePolicy","CookieConsent")">cookies page</a>.</p>

```

Comment:
Could we add some margin above this paragraph, maybe `nhsuk-u-margin-top-3` class?

Reply:
Done